primitives/src/locktime/relative.rs

```diff
@@ -200,7 +200,7 @@ impl LockTime {
     ///
     /// ```rust
     /// # use bitcoin_primitives::Sequence;
-    /// # use bitcoin_primitives::locktime::relative::{Height, Time};
+    /// # use bitcoin_primitives::relative::{Height, Time};
     ///
     /// # let required_height = 100;       // 100 blocks.
     /// # let intervals = 70;     // Approx 10 hours.
```

Comment:
```suggestion
    /// # let intervals = 70;     // Approx 10 hours.
    /// # let current_height = || relative::Height::from(required_height + 10);
    /// # let current_time = || relative::Time::from_512_second_intervals(intervals + 10);
```

It is the addition of `relative::` in front of `Height` and `Time` that is needed so that when looking at the code it is clear whether it is a relative or absolute height or time.

Reply:
Aha. Got you.